assignments/assignment05.rb

```diff
@@ -6,6 +6,42 @@
 #  Problem 1 -  implement prop_reader, write MiniTest unit tests
 
 # expected results:
+
+require 'minitest/autorun'
+
+class Class
+	#prop_reader :flavor
+
+	def prop_reader(*flavors)
+		flavors.each do |flavor|
+	    define_method flavor do
+	      instance_variable_get "@#{flavor}"
+	    end
+		end
+	end
+
+  def initialize(flavor)
```

Comment:
this definitely doesn't belong here. Remember this is in class `Class`. Overriding the initialize method for Class could have some seriously bad consequences.